apps/web/package.json

```diff
@@ -61,6 +72,8 @@
     "recharts": "^2.12.7",
     "satori": "^0.10.14",
     "sharp": "^0.33.4",
+    "three": "^0.168.0",
+    "three-stdlib": "^2.33.0",
```

Comment:
double checking all these deps ended up getting used and we def want them in the new version

I know sometimes I add a lib and decide against it after working with it for a bit

`maath` looks sus to me when I look it up on npm, since there's no repo, docs, or readme

Reply:
yeah we need those. not sure about maath cc @zencephalon ?

Reply:
all other additions good to merge?

Reply:
yeah